### ISSUE TITLE
Question about plugin maintenance

### ISSUE THREAD

**Issue by ivangonzalezacuna** [K-Phoen/backstage-plugin-grafana#78]
In our company we have been using this plugin for quite some time already. The work done here was pretty good so far, but we have noticed that it is not being maintained that often anymore (the last release was in March, already 6 months ago) and some of the features we wanted to have, which might be really useful for others as well, are not merged at all.

Due to this situation, we ended up creating a fork of this project, add our suggestions to it and keep all the backstage dependencies up to date, to avoid any possible issue. However, we would like to get rid of our forked repo and use the actual plugin again.

I've already mentioned it in the Backstage's discord channel, and we would like to know if this plugin is going to be maintained in the near future or not. Otherwise, it might be a good idea if somebody decides to get the ownership of this plugin and bring it to life again. I was also thinking about adding it directly to the list of open-sourced plugins maintained by the backstage community.

What do you think about it?

**Comment by K-Phoen** [comment]
Hey there!

Happy to know that this humble side-project has proven useful to your company :)

Before I comment on maintenance questions, here's some context around this repository.

I created this plugin to play with both Grafana and Backstage. At the time, I was also using Backstage professionally, so it was easy to find reasons and time to maintain this repo. Since then, I changed jobs and no longer work on a Backstage deployment (directly at least).

That explains the delays and lack of regularity in updates & PR reviews: the project reverted to being a side-project only, work on it pretty much happening when/if I have some motivation and time.

However, the company you work for (and others I presume) understandably have different needs.

I can think of two main ways to ensure that these needs will be met:
* a fork: the license allows it, and it's the best way for a company to ensure that the work they need is done, when they need it to be done. There might even be people from the community is willing to do it for free
* some kind of discussion + sponsoring: as a way to offload maintenance/feature work to the original maintainer, while providing a fair incentive

So to summarize: I view this project as a side-project and I'll work on it from time to time with no promises on any regularity (unless some kind of sponsoring is involved).

Again: I'm genuinely happy to learn that this repo is useful enough for your company to even have this discussion (and kinda surprised, if I'm honest)

**Comment by ivangonzalezacuna** [comment]
Thanks for your answer @K-Phoen. I would say the fork is definitely the best option, specially because another person is already willing to maintain it (@Vity01). So if you agree, we could start with the transfer of the ownership some time soon, and bring this project back to track :smiley:

**Comment by Vity01** [comment]
@K-Phoen  what about to just assign multiple maintainers for this existing repo without necessary forking?

**Comment by K-Phoen** [comment]
A fork sounds good to me :)

I like the idea of retaining ownership on this repo since I still plan on playing with it from time to time.

**Comment by jtreher** [comment]
Could we also move this into the main Backstage repo and "donate" it?

**Comment by ivangonzalezacuna** [comment]
> Could we also move this into the main Backstage repo and "donate" it?

That was one of our initial ideas, but I think that's not an option anymore: https://github.com/backstage/backstage/issues/20266
But anyways, creating a fork and maintaining it sounds like the best approach here.

**Comment by ivangonzalezacuna** [comment]
Hey @Vity01 , will you then create the new repository for the plugin? Or should we find a different approach to publish it?
